src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -53,25 +53,24 @@ public void equals() {
         // different person -> returns false
         assertFalse(findFirstCommand.equals(findSecondCommand));
     }
-
-    // @Test
-    // public void execute_zeroKeywords_noPersonFound() {
-    //     String expectedMessage = String.format(MESSAGE_ENTRIES_LISTED_OVERVIEW, 0);
-    //     NameContainsKeywordsPredicate predicate = preparePredicate(" ");
-    //     FindCommand command = new FindCommand(predicate);
-    //     expectedModel.updateFilteredPersonList(predicate);
-    //     assertCommandSuccess(command, model, expectedMessage, expectedModel);
-    //     assertEquals(Collections.emptyList(), model.getFilteredPersonList());
-    // }
-    // @Test
-    // public void execute_multipleKeywords_multiplePersonsFound() {
-    //     String expectedMessage = String.format(MESSAGE_ENTRIES_LISTED_OVERVIEW, 3);
-    //     NameContainsKeywordsPredicate predicate = preparePredicate("Kurz Elle Kunz");
-    //     FindCommand command = new FindCommand(predicate);
-    //     expectedModel.updateFilteredPersonList(predicate);
-    //     assertCommandSuccess(command, model, expectedMessage, expectedModel);
-    //     assertEquals(Arrays.asList(CARL, ELLE, FIONA), model.getFilteredPersonList());
-    // }
+    @Test
+    public void execute_zeroKeywords_noPersonFound() {
```

Comment:
Should we rename this to `execute_zeroKeywords_noEntryFound`?

Reply:
yeap youre right, ill change it with my next commits!